src/Sarif/NotYetAutoGenerated/SarifRewritingVisitor.cs

```diff
@@ -720,11 +709,8 @@ public virtual Run VisitRun(Run node)
                         node.VersionControlProvenance[index_0] = VisitNullChecked(node.VersionControlProvenance[index_0]);
                     }
                 }
-
-                // OriginalUriBaseIds are directories, not files. We'll disable this visit until the
-                // schema can catch up with this reality.
```

Comment:
Don't remove this comment. It's important to know why files are in the `NotYetAutogenerated` directory.